### PR TITLE
1.5 The integration of genearl json and rest

### PR DESCRIPTION
**Pull request by yamicro**
The integration of genearl json and rest

### CODE REVIEW

**Review by EnableAsync — CHANGES_REQUESTED** [review]
.DS_Store can be removed.

File: general/jsonrpc/go-client/app/user.go [hunk 1]
```diff
@@ -95,3 +95,15 @@ type UserProvider2 struct {
 func (u *UserProvider2) Reference() string {
 	return "UserProvider2"
 }
+
+func (UserProvider) JavaClassName() string {
+	return "com.ikurento.user.UserProvider"
+}
+
+func (UserProvider1) JavaClassName() string {
+	return "com.ikurento.user.UserProvider"
```
Comment (AlexStocks) [inline general/jsonrpc/go-client/app/user.go:104]: com.ikurento.user.UserProvider1？

File: general/jsonrpc/go-client/app/user.go [hunk 2]
```diff
@@ -95,3 +95,15 @@ type UserProvider2 struct {
...
+}
+
+func (UserProvider2) JavaClassName() string {
+	return "com.ikurento.user.UserProvider"
```
Comment (AlexStocks) [inline general/jsonrpc/go-client/app/user.go:108]: com.ikurento.user.UserProvider2？

File: general/jsonrpc/go-server/tests/integration/main_test.go [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+func (UserProvider) JavaClassName() string {
+	return "com.ikurento.user.UserProvider"
+}
+
+func (UserProvider1) JavaClassName() string {
+	return "com.ikurento.user.UserProvider"
+}
+
+func (UserProvider2) JavaClassName() string {
+	return "com.ikurento.user.UserProvider"
```
Comment (AlexStocks) [inline general/jsonrpc/go-server/tests/integration/main_test.go:131]: com.ikurento.user.UserProvider2？

File: general/jsonrpc/go-server/tests/integration/main_test.go [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+func (u *UserProvider2) Reference() string {
+	return "UserProvider2"
+}
+
+func (UserProvider) JavaClassName() string {
+	return "com.ikurento.user.UserProvider"
+}
+
+func (UserProvider1) JavaClassName() string {
+	return "com.ikurento.user.UserProvider"
```
Comment (AlexStocks) [inline general/jsonrpc/go-server/tests/integration/main_test.go:127]: com.ikurento.user.UserProvider1？

Comment (yamicro) [reply general/jsonrpc/go-server/tests/integration/main_test.go:127]: I imitate the previous code.In the previous code three UserProvider use the same interface name in configuration file.So I user the com.ikurento.user.UserProvider.

**Pull request closed without merging**